config-communicate.json

```diff
@@ -111,9 +120,79 @@
         "UIFeature.showEncryptionSettings": false,
         "UIFeature.showSimplifiedVersionInformation": true,
         "UIFeature.showAdvancedAboutInformation": false,
-        "MessageComposerInput.showStickersButton": false
+        "MessageComposerInput.showStickersButton": false,
+        "UIFeature.showSecurityKeyBackupPrompt": false,
+        "UIFeature.showLiberateToCommunicateText": false,
+        "UIFeature.secondaryLogoUrl": ["themes/element/img/logos/element-logo-secondary.png"]
+
     },
-    "jitsi": {
-        "preferredDomain": "jitsi.riot.im"
-    }
-}
\ No newline at end of file
+    "DISABLED_jitsi": {
+        "preferredDomain": null
+    },
+    "terms_and_conditions_links": [
```

Comment:
Please remove user settings changes in this file

Reply:
This action has been taken.